fileparser.py

```diff
@@ -1,29 +1,23 @@
-'''
-fileparser.py - contains basic functions to perform file operations
-'''
-
-class fo:
+class fp:
```

Comment:
Thoughts on changing the class name to `file_parser` just to be more explicit?

Reply:
I like that - but then perhaps we could call the script `fp.py`? My worry is that `fileparser.file_parser` would be a little less clear since it doesn't delineate the differences between the two as well, but perhaps `fp.file_parser` would?